### PR TITLE
Личный проект: большой взрыв

### PR DESCRIPTION
**Pull request by grigorievk**
Start for project environment and add basic functionality.

---
:mortar_board: [Личный проект: большой взрыв](https://up.htmlacademy.ru/ecmascript/9/user/1075187/tasks/6)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,15 @@
   "description": "Личный проект «Большое путешествие» от HTML Academy",
   "scripts": {
     "eslint": "eslint src/**/*.js",
-    "test": "npm run eslint"
+    "test": "npm run eslint",
+    "build": "webpack",
+    "start": "webpack-dev-server --open"
```
Comment (rnskv) [inline package.json]: I think that be better if you move "open" property to the devserver configuration (open: true)

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const path = require(`path`);
+
+module.exports = {
+  mode: `development`,
+  entry: `./src/main.js`,
```
Comment (rnskv) [inline webpack.config.js]: Use path.resolve and __dirname

Comment (grigorievk) [reply webpack.config.js]: Are you sure? For entry point? Sorry, I read again the webpack documentation and did not understand why I should do this.

File: package.json (hunk 1 above)
Comment (grigorievk) [reply package.json]: Ok, I did it.

File: webpack.config.js (hunk 2 above)
Comment (rnskv) [reply webpack.config.js]: Thank you for your comment! I'm sorry, its wrong

**Pull request merged**